playground/backend/cmd/server/controller.go

```diff
@@ -87,19 +87,48 @@ func (controller *playgroundController) CheckStatus(ctx context.Context, info *p
 	return &pb.CheckStatusResponse{Status: status}, nil
 }
 
-//GetRunOutput is returning output of execution for specific pipeline by PipelineUuid
+// GetRunOutput is returning output of execution for specific pipeline by PipelineUuid
 func (controller *playgroundController) GetRunOutput(ctx context.Context, info *pb.GetRunOutputRequest) (*pb.GetRunOutputResponse, error) {
 	pipelineId := info.PipelineUuid
+	runOutputLastIndex, err := controller.cacheService.GetValue(ctx, uuid.MustParse(pipelineId), cache.RunOutputIndex)
+	if err != nil {
+		logger.Errorf("%s: GetRunOutput(): cache.GetValue: error: %s", pipelineId, err.Error())
+		return nil, errors.NotFoundError("GetRunOutput", "there is no run output for pipelineId: "+pipelineId+", subKey: cache.RunOutputLastIndex")
```

Comment:
why we can't use the standard format way without string concatenation?

Reply:
Is this question still actual?